src/net/client/SyncHttp.cpp

```
@@ -0,0 +1,210 @@
+  } catch (...) {
+  }
+  return total;
+}
+
+bool
+Perform(CurlEasy &easy, StringOutputStream &body, SyncHttpResponse &response,
+        HeaderCapture *headers, SyncHttpProgressFn progress) noexcept
+{
+  response = {};
+
+  easy.SetWriteFunction(WriteToStream, &body);
```

Cppcheck (2.21.0)

[error] Unhandled exception thrown in function declared not to throw exceptions.

(throwInNoexceptFunction)